forte/processors/data_augment/algorithms/eda_processors.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Data augmentation processors from the paper "EDA: Easy Data Augmentation
+Techniques for Boosting Performance on Text Classification Tasks", including
+Random Swap, Random Insertion and Random Deletion. All three processors are
+implemented based on the ReplacementDataAugmentProcessor.
+"""
+
+import random
+from typing import List, Tuple, Dict
+from nltk.corpus import stopwords
+from math import ceil
+
+from forte.data.data_pack import DataPack
+from forte.data.multi_pack import MultiPack
+from forte.processors.base import ReplacementDataAugmentProcessor
+from forte.utils.utils import get_class, create_class_with_kwargs
+
+__all__ = [
+    "RandomSwapDataAugmentProcessor",
+    "RandomInsertionDataAugmentProcessor",
+    "RandomDeletionDataAugmentProcessor",
+]
+
+
+class RandomSwapDataAugmentProcessor(ReplacementDataAugmentProcessor):
+    r"""
+    Data augmentation processor for the Random Swap operation.
+    Randomly choose two words in the sentence and swap their positions.
+    Do this n times.
```

Comment:
How do we specify `n`?

Reply:
`n` is replaced by `alpha`. I will update the documentation.